src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificatesTabCoordinator.swift

```diff
@@ -138,7 +138,8 @@ final class HealthCertificatesTabCoordinator {
 						}
 					)
 				)
-			}
+			},
+			showAlertAfterRegroup: showAlertAfterRegrouping
```

Comment:
Passing the function to the closure parameter will capture HealthCertificatesTabCoordinator strongly inside of HealthCertificateOverviewViewController. 
I think it will lead to a circular dependency.

Reply:
added already a weak self